src/editor/Collaboration.js

```diff
@@ -53,6 +62,11 @@ define(function (require, exports, module) {
             case "codemirror-change":
                 _handleCodemirrorChange(msg.payload);
                 break;
+            case "file-rename":
+                var oldPath = Path.relative(msg.payload.oldPath, root);
```

Comment:
we use msg.payload a lot here. Let's reduce duplication:
```js
var payload = msg.payload;
var oldPath, newPath;

switch(msg.type) {
    case ...;
    case "file-rename":
        oldPath = Path.relative(payload.oldPath, root);
        newPath = Path.relative(payload.newPath, root);
        console.log("Renamed ", oldPath, " to ", newPath);
        break;
    case ...;
}
```
and change every reference to the `payload` var to use the one you create above